kern/vmx.c

```diff
@@ -610,7 +610,7 @@ static unsigned long segment_base(u16 selector)
 	v = get_desc_base(d);
 #ifdef CONFIG_X86_64
        if (d->s == 0 && (d->type == 2 || d->type == 9 || d->type == 11))
-               v |= ((unsigned long)((struct ldttss_desc64 *)d)->base3) << 32;
+               v |= ((unsigned long)((ldt_desc *)d)->base3) << 32;
```

Comment:
just a typedef. The compiler was complaining about an incomplete pointer type, and tbh I didn't look into where this may have changed, but I believe the actual struct definition is just hidden from the include tree in this file now.